docsite/source/commands-with-subcommands-&-params.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Commands with subcommands and params
+layout: gem-single
+name: dry-cli
+---
+
+There is a way to register command with arguments, options and subcommands.
+
+This helps to implement complex nested logic.
+
+Arguments and options can be defined for parent both commands and child commands.
+
+**Note:** If you call a command with an argument equal to the name of the subcommand, it will call the subcommand instead of the parent command.
```

Comment:
```suggestion
^INFO
If you call a command with an argument equal to the name of the subcommand, it will call the subcommand instead of the parent command.
^
```